src/applications/vaos/referral-appointments/components/PendingReferralCard.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import AppointmentFlexGrid from '../../appointment-list/components/AppointmentsPage/AppointmentFlexGrid';
+import ListItem from '../../appointment-list/components/AppointmentsPage/ListItem';
+import AppointmentRow from '../../appointment-list/components/AppointmentsPage/AppointmentRow';
+import AppointmentColumn from '../../appointment-list/components/AppointmentsPage/AppointmentColumn';
+
+export default function PendingReferralCard({ referral }) {
```

Comment:
Not sure if this is a card component. It's more of a row layout to me since it defines how the columns are laid out within the row. The existing components in this view are of type 'RequestedAppointmentsListGroup' (I'm not entirely happy with this name either). I would be inclined to have these new list items be of this type. This will also address the layout issue when displayed full screen.

![Screenshot 2024-07-23 at 10 20 43 AM](https://github.com/user-attachments/assets/c5294cff-1df2-46b0-ad73-46c868d8c752)
---
![localhost_3001_my-health_appointments_pending (3)](https://github.com/user-attachments/assets/35d09df2-e35d-41a5-bfe4-b185c1e25ded)